packages/url_launcher/url_launcher_platform_interface/test/url_launcher_platform_test.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:url_launcher_platform_interface/url_launcher_platform_interface.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  // TODO(hterkelsen): Re-enable this test once we land the corresponding
+  //     change in package:url_launcher
+  group('UrlLauncher with mock platform', () {
+    final MockUrlLauncherPlatform platform = MockUrlLauncherPlatform();
+    // Skip until we add this to package:url_launcher
+    //urlLauncherPlatform = platform;
+
+    test('can mock "canLaunch"', () async {
+      expect(canLaunch('http://www.google.com'), completion(isFalse));
+
+      platform.launchableUrls.add('http://www.google.com');
+
+      expect(canLaunch('http://www.google.com'), completion(isTrue));
+
+      platform.launchableUrls.clear();
+    });
+
+    test('can mock "launch"', () async {
+      expect(launch('http://www.google.com'), completion(isFalse));
+
+      platform.launchableUrls.add('http://www.google.com');
+
+      expect(launch('http://www.google.com'), completion(isTrue));
+
+      platform.launchableUrls.clear();
+    });
+
+    test('can mock "closeWebView"', () async {
+      expect(closeWebView(), completes);
+    });
+  }, skip: true);
+}
+
+class MockUrlLauncherPlatform extends UrlLauncherPlatform {
```

Comment:
Maybe extract this to its own test/mocks/mock_url_launcher_platform.dart file?

Reply:
Removed this test. Just testing the `MethodChannel` implementation now